package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "bootstrap-wysiwyg",
+  "version": "1.0.3-beta",
```

Comment:
You mean that it'll be harder to maintain the version number in two different files?

Perhaps you could create a new gulp task with [gulp-bump](https://www.npmjs.com/package/gulp-bump) to bump versions from bower.json and package.json simultaneously.